cypress/integration/racingcar.spec.js

```diff
@@ -3,143 +3,249 @@ describe('자동차 경주 게임', () => {
-    cy.get('#car-names-input').should('be.visible');
-    cy.get('#car-names-submit').should('be.visible');
+  describe('자동차 이름 입력창에 대한 테스트', () => {
+    it('자동차 이름을 입력하지 않으면 경고창 메시지를 보여준다.', () => {
+      // given
+      const alertStub = cy.stub();
+      cy.on('window:alert', alertStub);
+
+      cy.get('#car-names-input').clear();
+
+      // when
+      // then
+      cy.get('#car-names-submit')
+        .click()
+        .then(() => {
+          expect(alertStub).to.be.calledWith('자동차 이름은 최소 1글자에서 최대 5글자까지 입력해주세요.');
+        });
+    });
+
+    it('자동차 이름이 6글자 이상이면 경고창 메시지를 보여준다.', () => {
+      // given
+      const alertStub = cy.stub();
+      cy.on('window:alert', alertStub);
+
+      cy.get('#car-names-input').type('EVERYDAY');
+
+      // when
+      // then
+      cy.get('#car-names-submit')
+        .click()
+        .then(() => {
+          expect(alertStub).to.be.calledWith('자동차 이름은 최소 1글자에서 최대 5글자까지 입력해주세요.');
+        });
+    });
   });
 
-  it('자동차 이름을 입력하지 않으면 경고창 메시지를 보여준다.', () => {
-    const alertStub = cy.stub();
-    cy.on('window:alert', alertStub);
-
-    cy.get('#car-names-input').clear();
-    cy.get('#car-names-submit')
-      .click()
-      .then(() => {
-        expect(alertStub).to.be.called;
-      });
-  });
-
-  it('자동차 이름이 6글자 이상이면 경고창 메시지를 보여준다.', () => {
-    const alertStub = cy.stub();
-    cy.on('window:alert', alertStub);
-
-    cy.get('#car-names-input').type('EVERYDAY');
-    cy.get('#car-names-submit')
-      .click()
-      .then(() => {
-        expect(alertStub).to.be.called;
-      });
+  describe('시도 횟수에 대한 테스트', () => {
+    it('시도 횟수 입력창은 자동차 이름을 제출한 후에 표시된다.', () => {
+      // given
+      cy.get('#racing-try-count').should('not.be.visible');
+
+      // when
+      cy.get('#car-names-input').type('CHILL,HIP');
+      cy.get('#car-names-submit').click();
+
+      // then
+      cy.get('#racing-try-count').should('be.visible');
+    });
+
+    it('시도 횟수가 0 이하일 경우 경고창을 호출한다.', () => {
+      // given
+      cy.get('#racing-try-count').should('not.be.visible');
+      cy.get('#car-names-input').type('CHILL,HIP');
+      cy.get('#car-names-submit').click();
+      cy.get('#racing-try-count').should('be.visible');
+
+      // when
+      const WRONG_INPUT_UNDER_ZERO = -1;
+      cy.get('#try-count-input').type(WRONG_INPUT_UNDER_ZERO);
+
+      const alertStub = cy.stub();
+      cy.on('window:alert', alertStub);
+
+      // then
+      cy.get('#try-count-submit')
+        .click()
+        .then(() => {
+          expect(alertStub).to.be.calledWith('시도 횟수는 1번 이상, 10번 이하여야 합니다.');
+        });
+    });
+
+    it('시도 횟수가 10 보다 크면 경고창을 호출한다.', () => {
+      // given
+      cy.get('#racing-try-count').should('not.be.visible');
+      cy.get('#car-names-input').type('CHILL,HIP');
+      cy.get('#car-names-submit').click();
+      cy.get('#racing-try-count').should('be.visible');
+
+      // when
+      const WRONG_INPUT_OVER_TEN = 11;
```

Comment:
```suggestion
      const WRONG_INPUT_UNDER_ZERO = TRY_COUNT_MIN_VALUE - 1;
      cy.get('#try-count-input').type(WRONG_INPUT_UNDER_ZERO);

      const alertStub = cy.stub();
      cy.on('window:alert', alertStub);

      // then
      cy.get('#try-count-submit')
        .click()
        .then(() => {
          expect(alertStub).to.be.calledWith('시도 횟수는 1번 이상, 10번 이하여야 합니다.');
        });
    });

    it('시도 횟수가 10 보다 크면 경고창을 호출한다.', () => {
      // given
      cy.get('#racing-try-count').should('not.be.visible');
      cy.get('#car-names-input').type('CHILL,HIP');
      cy.get('#car-names-submit').click();
      cy.get('#racing-try-count').should('be.visible');

      // when
      const WRONG_INPUT_OVER_TEN = TRY_COUNT_MAX_VALUE + 1
```
임계값은 설정해주신 상수를 활용해서 테스트하는게 더 유의미하다고 생각되네요 👀